requirements.txt

```diff
@@ -1,8 +1,8 @@
-django >= 1.9, < 1.10
+django >= 1.10, < 1.11
 xlrd == 1.0.0
 xlwt == 1.1.2
 django-fsm == 2.4.0
-django-webtest == 1.8.0
+django-webtest >= 1.8.0
```

Comment:
why do you allow all later versions (including major and minor version changes) here but not anywhere else?